notebooks/rag.ipynb

```diff
@@ -1045,105 +1048,20 @@
    "metadata": {
     "tags": []
    },
-   "outputs": [
-    {
-     "name": "stdout",
-     "output_type": "stream",
-     "text": [
-      "DROP TABLE\n",
-      "CREATE TABLE\n",
-      "SET\n",
-      "SET\n",
-      "SET\n",
-      "SET\n",
-      "SET\n",
-      " set_config \n",
-      "------------\n",
-      " \n",
-      "(1 row)\n",
-      "\n",
-      "SET\n",
-      "SET\n",
-      "SET\n",
-      "SET\n",
-      "ALTER TABLE\n",
-      "ALTER TABLE\n"
-     ]
-    },
-    {
-     "name": "stderr",
-     "output_type": "stream",
-     "text": [
-      "psql:/efs/shared_storage/goku/sql_dumps/gte-base_300_50.sql:20: ERROR:  relation \"public.data_document\" does not exist\n",
-      "psql:/efs/shared_storage/goku/sql_dumps/gte-base_300_50.sql:22: ERROR:  relation \"public.data_document\" does not exist\n"
-     ]
-    },
-    {
-     "name": "stdout",
-     "output_type": "stream",
-     "text": [
-      "DROP SEQUENCE\n",
-      "DROP TABLE\n"
-     ]
-    },
-    {
-     "name": "stderr",
-     "output_type": "stream",
-     "text": [
-      "psql:/efs/shared_storage/goku/sql_dumps/gte-base_300_50.sql:25: ERROR:  sequence \"data_document_id_seq\" does not exist\n",
-      "psql:/efs/shared_storage/goku/sql_dumps/gte-base_300_50.sql:26: ERROR:  table \"data_document\" does not exist\n"
-     ]
-    },
-    {
-     "name": "stdout",
-     "output_type": "stream",
-     "text": [
-      "DROP EXTENSION\n",
-      "CREATE EXTENSION\n",
-      "COMMENT\n",
-      "SET\n",
-      "SET\n",
-      "CREATE TABLE\n",
-      "ALTER TABLE\n",
-      "CREATE SEQUENCE\n",
-      "ALTER SEQUENCE\n",
-      "ALTER SEQUENCE\n",
-      "CREATE TABLE\n",
-      "ALTER TABLE\n",
-      "CREATE SEQUENCE\n",
-      "ALTER SEQUENCE\n",
-      "ALTER SEQUENCE\n",
-      "ALTER TABLE\n",
-      "ALTER TABLE\n",
-      "COPY 40433\n",
-      "COPY 32276\n",
-      " setval \n",
-      "--------\n",
-      "  40433\n",
-      "(1 row)\n",
-      "\n",
-      " setval \n",
-      "--------\n",
-      "  32276\n",
-      "(1 row)\n",
-      "\n",
-      "ALTER TABLE\n",
-      "ALTER TABLE\n",
-      " count \n",
-      "-------\n",
-      " 32276\n",
-      "(1 row)\n",
-      "\n"
-     ]
-    }
-   ],
+   "outputs": [],
    "source": [
     "%%bash\n",
-    "# Drop table and load index\n",
+    "# Drop the existing `document` table and create a new one with the schema to store embeddings. \n",
     "psql \"$DB_CONNECTION_STRING\" -c \"DROP TABLE IF EXISTS document;\"  # drop\n",
     "sudo -u postgres psql -f $MIGRATION_FP  # create\n",
-    "psql \"$DB_CONNECTION_STRING\" -f $SQL_DUMP_FP  # load\n",
```

Comment:
`$SQL_DUMP_FP` will only be created after the embeddings are written into Postgres in the following step, as shown below. Hence, we don't have the file `$SQL_DUMP_FP` at this moment.

```sh
sudo -u postgres pg_dump -c > $SQL_DUMP_FP  # save
```
![Screen Shot 2024-03-26 at 12 25 00 AM](https://github.com/ray-project/llm-applications/assets/20109646/c37f860b-123d-4924-aa70-d15413ce0957)